scripts/cti

```diff
@@ -155,7 +155,7 @@ while (( "$#" )); do
       shift 2
       ;;
     -M|--master)
```

Comment:
i know we are on track to deprecate CT and cti. until it is deprec, it will continue to cause confusion. as Zekun suggested, let's clean up the master ref.  ie rename --master option to --stable.